src/components/organisms/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { VFC } from 'react';
+import styled from 'styled-components';
+import { colors } from 'src/styles/Tokens';
+
+type BreadcrumbsProps = {
+  items: {
+    link_url?: string;
+    text: string;
+  }[]
+}
+
+const Wrapper = styled.div`
+  width: 100%;
+  font-size: 12px;
+  background-color: ${colors.YumeWhiteGreen2};
+  box-shadow: inset 0px 1px 4px rgba(0, 0, 0, 0.25);
+  ul {
+    padding: 0.5em;
+    justify-content: start;
+    display: flex;
+    list-style: none;
+  }
+  li {
+    position: relative;
+  }
+  li > a {
+    display: block;
+    padding: 0.2em 0.5em;
+    background-color: ${colors.White};
+    box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 4px;
+  }
```

Comment:
[imo] このようにネストして書くこともできます。

```suggestion
  li {
    position: relative;
    a {
      display: block;
      padding: 0.2em 0.5em;
      background-color: ${colors.White};
      box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.25);
      border-radius: 4px;
    }
  }
```